llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -5496,6 +5496,9 @@ const char* AMDGPUTargetLowering::getTargetNodeName(unsigned Opcode) const {
   NODE_NAME_CASE(LDS)
   NODE_NAME_CASE(FPTRUNC_ROUND_UPWARD)
   NODE_NAME_CASE(FPTRUNC_ROUND_DOWNWARD)
+  NODE_NAME_CASE(READLANE)
+  NODE_NAME_CASE(READFIRSTLANE)
```

Comment:
Add these to `AMDGPUTargetLowering::isSDNodeAlwaysUniform`

Reply:
done